lib/_tokenUtils.ts

```diff
@@ -25,29 +25,27 @@ export const getCookie = (name: string): string => {
   return "";
 };
 
-export const ANONYMOUS_USER_TOKEN = "ANONYMOUS_USER_TOKEN";
-
-export function setUserToken(userToken: string | number): void {
-  if (userToken === ANONYMOUS_USER_TOKEN) {
-    if (!supportsCookies()) {
-      throw new Error(
-        "Tracking of anonymous users is only possible on environments which support cookies."
-      );
-    }
-    const foundToken = getCookie(COOKIE_KEY);
-    if (
-      !foundToken ||
-      foundToken === "" ||
-      foundToken.indexOf("anonymous-") !== 0
-    ) {
-      this._userToken = `anonymous-${createUUID()}`;
-      setCookie(COOKIE_KEY, this._userToken, this._cookieDuration);
-    } else {
-      this._userToken = foundToken;
-    }
+export function setAnonymousUserToken(): void {
+  if (!supportsCookies()) {
+    throw new Error(
+      "Tracking of anonymous users is only possible on environments which support cookies."
+    );
+  }
+  const foundToken = getCookie(COOKIE_KEY);
+  if (
+    !foundToken ||
+    foundToken === "" ||
+    foundToken.indexOf("anonymous-") !== 0
+  ) {
+    this._userToken = `anonymous-${createUUID()}`;
+    setCookie(COOKIE_KEY, this._userToken, this._cookieDuration);
   } else {
-    this._userToken = userToken;
+    this._userToken = foundToken;
```

Comment:
Is it expected that we don't call `onUserTokenChangeCallback` anymore with the anonymous token?

Reply:
That's a good point that I missed. I'll fix it 👍🏼

Reply:
https://github.com/algolia/search-insights.js/pull/236/commits/461c9965347af64179de83a78ecaa9cfec2651e1